.github/workflows/sign-and-release-to-github.yml

```diff
@@ -2,15 +2,56 @@ name: continuous prerelease to GitHub
 
 on:
   push:
-    branches: [ main ]
-
+    branches: [ "*" ]
   workflow_dispatch:
 
 jobs:
-  sign-tag-release:
+  prepare-deployment:
     runs-on: ubuntu-latest
+    outputs:
+      tag-slug: ${{ steps.determine-npm-tag.outputs.tag-slug }}
```

Comment:
Where does this come from? And it looks like we're not using it anywhere?

Reply:
Whoops! I based this on [the workflows I setup at my previous place](https://github.com/inrupt/solid-client-js/actions/workflows/cd.yml), and this is a remnant of that. Removed in 35bdb1a.